resources/js/components/WorksMap.vue

```diff
@@ -196,8 +196,12 @@ export default {
             properties: {
               id: work.id,
               title: work.name,
-              description: '',
-              url: work.url
+              architects: [
+                ..._.map(work.architects, 'full_name'),
+                ..._.map(work.architects, 'name')
+              ],
```

Comment:
Confused by this. Is this really what we want? A list of full_names followed by a list of names? Or am I reading it wrong?

Reply:
ouch! my bad.
should be 
```suggestion
              architects: [
                ..._.map(work.architects, 'full_name'),
                ..._.map(work.other_architects, 'name')
              ],
```